### DISCUSSION TITLE
More items to spend mythic raid points on

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#6920] [Suggestions]
I have 32 mythic points but nothing to spend them on because I need nothing from T4
however i still need a couple of items from SSC (Vashj Belt, Tsunami, Hydros Ring)
Let me buy at least some of the items or all of them
Make them increase cost and make it so you have to clear SSC a handful of times in order to even buy the items

**Comment by BeefSteaker** [comment]
Unfortunately mythic mode has been cancelled

tokens are specific to each tier anyway, so any current tokens you have wouldn't be useable for SSC tier stuff

**Reply by Never6** [reply]
im aware that it was cancelled i guess i can just direct people here then https://github.com/chromiecraft/chromiecraft/discussions/6894
